src/main/fc/fc_msp.c

```diff
@@ -2145,8 +2144,31 @@ void mspFcProcessReply(mspPacket_t *reply)
     UNUSED(src); // potentially unused depending on compile options.
 
     switch (reply->cmd) {
-        case MSP_DISPLAYPORT: {
+#ifndef OSD_SLAVE
+    case MSP_ANALOG:
+        {
```

Comment:
Just a detail, this formating is a bit wasteful. The block is only there to allow local variables, there is no need for new indentation level.

What about:
``` 
switch (reply->cmd) {
case MSP_ANALOG: {
    uint8_t batteryVoltage = sbufReadU8(src);
    ...
    break;
}
```
But I'm not sure about closing bracket, maybe it should be indented on same level as `break;`